README.md

```diff
@@ -28,6 +30,147 @@ git clone git@github.com:kamp-us/monorepo.git
 cd monorepo
 ```
 
+- Install dependencies.
+
+```sh
+npm install
```

Comment:
Since we're installing dependencies from scratch and we have `package-lock.json`, it's more faster and cleaner way to install packages via `npm ci`

```suggestion
npm ci
```